src/gui/layertree/qgslayertreeview.h

```diff
@@ -447,6 +448,7 @@ class GUI_EXPORT QgsLayerTreeView : public QTreeView
 
     bool mShowPrivateLayers = false;
 
+    QTimer *mDoubleClickTimer = nullptr;
```

Comment:
```suggestion
    QTimer *mBlockDoubleClickTimer = nullptr;
```

Or am I reading this wrong?